build.assets/tooling/cmd/please-run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env sh
+
+# This script is a helper that tells developers what generated content is out of date
+# and which command to run.
+# When running on GitHub actions, the script will also create an error in the PR and
+# collapse the diff to improve readability.
+
+set -eu
+
```

Comment:
Suggestion: wrap script body in a function, particularly if you take my suggestion about the echo_gha function below it makes things more organized.

Eg:


```shell
set -eu

echo_gha() {
  # ...
}

main() {
  # script body goes here
}

main "$@"
```